lib/bolt_server/transport_app.rb

```diff
@@ -257,7 +257,10 @@ def modulepath_from_environment(environment_name)
     end
 
     def in_pe_pal_env(environment)
-      return [400, '`environment` is a required argument'] if environment.nil?
+      if environment.nil?
+        err = Bolt::ValidationError.new("'environment' is a required argument")
```

Comment:
If we added an `error Bolt::ValidationError` handler, we could just raise this error normally. Although it looks like there are a couple of places we _currently_ raise ValidationError that actually represent misconfigurations of bolt-server itself and properly should return 500 (they do now), so those would probably also need to change.